### PR TITLE
small improvement on gc reporting

### PR DESCRIPTION
**Pull request by hermeGarcia**
### Description
Better logs for garbage collection.

### How was this PR tested?
Local tests

### CODE REVIEW

File: nucliadb_node/src/writer/mod.rs [hunk 1]
```diff
@@ -294,12 +294,4 @@ impl NodeWriterService {
...
         ShardIds { ids }
     }
-
-    #[tracing::instrument(skip_all)]
-    pub fn paragraph_count(&self, shard_id: &ShardId) -> NodeResult<Option<u64>> {
-        let Some(shard) = self.get_shard(shard_id) else {
-            return Ok(None);
-        };
-        shard.paragraph_count().map(|v| Some(v as u64))
-    }
```
Comment (jotare) [inline nucliadb_node/src/writer/mod.rs:304]: Why are you removing this?

File: nucliadb_vectors/src/data_point_provider/mod.rs [hunk 2]
```diff
@@ -129,16 +129,19 @@ impl Index {
         use std::collections::HashSet;
         let work_flag = self.work_flag.try_to_start_working()?;
         let state = self.read_state();
-        let non_garbage_dp: HashSet<_> = state.dpid_iter().map(|s| s.to_string()).collect();
+        let in_use_dp: HashSet<_> = state.dpid_iter().collect();
```
Comment (jotare) [inline nucliadb_vectors/src/data_point_provider/mod.rs:132]: `in_use_dp` stands for "data points in use"?

File: nucliadb_node/src/writer/mod.rs (hunk 1 above)
Comment (hermeGarcia) [reply nucliadb_node/src/writer/mod.rs:304]: is not used.

File: nucliadb_vectors/src/data_point_provider/mod.rs (hunk 2 above)
Comment (hermeGarcia) [reply nucliadb_vectors/src/data_point_provider/mod.rs:132]: yes

**Review by jotare — APPROVED** [review]

**Pull request merged**